acme.go

```diff
@@ -116,7 +117,8 @@ type AuthzID struct {
 // Client implements ACME spec.
 type Client struct {
 	http.Client
-	Key *rsa.PrivateKey
+	// Key.Public() must be *rsa.PublicKey
+	Key crypto.Signer
```

Comment:
github.com/google/acme package has been imported as /x/crypto/acme, internal for now: https://github.com/golang/crypto/tree/master/acme/internal/acme, and I'm planning to mirror it here as public. There's a note about it in the [readme](https://github.com/google/acme#acme).

I'm working on making /x/crypt/acme public (e.g. see https://go-review.googlesource.com/q/owner:ddos@google.com+status:open). The best place for PRs is https://go-review.googlesource.com, tbh.